docs/PROJECT-CONFIG.md

```diff
@@ -80,7 +80,13 @@ The name of the package or URL to the package git repository. This field is curr
 
 ### `version` - string
 
-The version of the package to use. May be a tag, branch or SHA.
+The version of the package to use.
+| Literal | Behaviour | Example |
+|---------|-----------|---------|
+| valid semantic version range | Specifies a range of valid semantic versions | `"v1.0.*", "~v1.0.0", "^v1.0.0", "v1"` |
```

Comment:
The syntax how to specify version ranges isn't explained here. Could you add either a description or a link to one, please?